minimal-plugin/src/main/java/io/micronaut/gradle/MicronautMinimalApplicationPlugin.java

```diff
@@ -212,6 +213,10 @@ private void configureGoogleCloudFunctionRuntime(Project project, Project p, Dep
         });
 
         // Google Cloud Function requires shadow packaging
-        p.getPluginManager().apply(ShadowPluginSupport.SHADOW_PLUGIN);
+        try {
+            p.getPluginManager().apply(ShadowPluginSupport.SHADOW_PLUGIN);
```

Comment:
Same comment as in #424 : the `try-catch` looks suspicious. If you needed this, it hints at a bug which required fixing.